src/Server/EventBusExplorer.Server.API/Controllers/QueuesController.cs

```diff
@@ -73,7 +80,34 @@ public async Task<IActionResult> GetAsync([FromRoute] string name)
     public async Task<IActionResult> DeleteAsync(
         [FromRoute] string name)
     {
-        await _queueService.DeleteAsync(name);
+        await _eventBusManagementService.DeleteQueueAsync(name);
         return NoContent();
     }
+
+    /// <summary>
+    /// Peek messages in queue
+    /// </summary>
+    /// <param name="queueName">Queue name</param>
+    /// <param name="receiveMode">Receive mode</param>
+    /// <param name="subQueue">Sub queue to query</param>
+    /// <param name="fromSequenceNumber">(Optional) Fetch messages from this one</param>
+    /// <param name="cancellationToken">(Optional) Cancellation token to cancel the operation</param>
+    /// <response code="200">List of peeked messages</response>
+    [ProducesResponseType(typeof(GetMessagesResponse), StatusCodes.Status200OK)]
+    [HttpGet("{queueName}/messages")]
+    public async Task<IActionResult> PeekMessagesAsync(
```

Comment:
```suggestion
    public async Task<IActionResult> GetMessagesAsync(
```
Per essere coerenti con quanto sotto e agnostici rispetto alla receiveMode specificata.